src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/merchandise_return.yml

```diff
@@ -23,3 +24,27 @@ admin_merchandise_returns_filter:
     redirectRoute: admin_merchandise_returns_index
     _legacy_controller: AdminReturn
 #  _legacy_link: AdminReturn:submitFilterorder_return
+
+admin_order_returns_edit:
+  path: /{orderReturnId}/edit
+  methods: [ GET, POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\CustomerService\MerchandiseReturnController::editAction
+    _legacy_controller: AdminReturn
+    #    _legacy_link: AdminReturn:updateorder_return
+    #    _legacy_parameters:
```

Comment:
I guess the `_legacy_parameters` can be left uncommented they don't enable the page anyway but it helps with automatic parameters reformatting